### PR TITLE
Build column info

### PR DESCRIPTION
**Pull request by PixelJanitor**

### CODE REVIEW

File: app/templates/components/build-header.hbs [hunk 1]
```diff
@@ -1,9 +1,46 @@
...
+        {{build.baseBuild.branch}}
+      </code>
+    </div>
+    <div class="f6">
+      Build {{#link-to "organization.project.builds.build" build.project.organization.slug build.project.slug build.baseBuild.id class="text-mono"}}{{build.baseBuild.buildNumber}}{{/link-to}}
+      -
+      {{moment-from-now build.baseBuild.createdAt interval=60000}}
+    </div>
+  </div>
+  <div style="margin-left: -8px">
```
Comment (cadeParade) [inline app/templates/components/build-header.hbs]: Why this inline style?

File: app/templates/components/build-header.hbs [hunk 2]
```diff
@@ -1,9 +1,46 @@
...
+    <figure class="svg-container">
+      {{inline-svg "build-column-separation-icon"}}
+    </figure>
+  </div>
+  <div class="w-50 text-light text-weight-semibold p-2">
+    <div class="flex align-center no-wrap">
+      Changes from
+      {{#if build.commit}}
```
Comment (cadeParade) [inline app/templates/components/build-header.hbs]: When would there not be a commit?

File: app/styles/keratin/base/typography.scss [hunk 3]
```diff
@@ -68,6 +68,7 @@ samp {
   margin-bottom: 0;
   font-family: var(--keratin-monospace-font);
   font-size: calc(var(--keratin-body-font-size) - 4px);
+  font-weight: 400 !important;
```
Comment (cadeParade) [inline app/styles/keratin/base/typography.scss]: Important?

File: app/templates/components/build-header.hbs (hunk 1 above)
Comment (PixelJanitor) [reply app/templates/components/build-header.hbs]: It's a one off case for just this icon. No reason to create a class for this one liner

File: app/templates/components/build-header.hbs (hunk 2 above)
Comment (PixelJanitor) [reply app/templates/components/build-header.hbs]: /shrug This is logic that was brought from everywhere else that uses a PR or commit. I assumed it was the safest route by reflecting those other implementations

File: app/styles/keratin/base/typography.scss (hunk 3 above)
Comment (PixelJanitor) [reply app/styles/keratin/base/typography.scss]: These types of elements can be inside another element that might have a utility class on it. Utility classes utilize `!important`s, but we ALWAYS want our monospaced font elements to have a font weight of 400. `important` on this ensures that the surrounding utility classes don't change that.

File: app/templates/components/build-header.hbs (hunk 1 above)
Comment (cadeParade) [reply app/templates/components/build-header.hbs]: In 8th grade I learned never to use inline styles....😱

Comment (PixelJanitor) [reply app/templates/components/build-header.hbs]: Oh for sure. It's a long standing, unspoken rule that is misunderstood. If something only happened in one place for one element and is never shared, it's nbd to style it there with it's markup.

**Pull request merged**